modules/installation-osp-describing-cloud-parameters.adoc

```diff
@@ -0,0 +1,53 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+
+[id="installation-osp-describing-cloud-parameters_{context}"]
+= Describing OpenStack parameters for the installer
+
+The {product-title} installer relies on information in a `clouds.yaml` file to run. The file describes OpenStack configuration parameters, including the project name, log in information, and authorization service URLs.
```

Comment:
Can credentials be provided while running the installer instead?

Reply:
@tomassedovic @iamemilio ^^^

Apparently, providing credentials to the installer is preferred to other methods. Is this the best/only way to accomplish this in Shiftstack flows?

Reply:
If I am not mistaken, a `clouds.yaml` file is **required**